chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestDefinition.java

```diff
@@ -98,9 +117,9 @@ public Builder withValidators(List<Class<? extends ResponseValidator>> validator
         }
 
         /**
-         * As one may expect, creates the object of {@link JLoadTest} type with custom parameters.
+         * Creates the object of JTestDefinition type with custom parameters.
```

Comment:
doxygen (who is generating our docu) is working better without links. He is generating links to the class names without the link tag

Idea is handling your case perfectly. Refactor->Rename will do the job